package.json

```diff
@@ -107,6 +107,7 @@
     "@rollup/plugin-commonjs": "^17.1.0",
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^11.2.0",
+    "@types/mocha": "^8.2.1",
```

Comment:
Let's exclude tests for now and cover only library code

Reply:
Actually checking the tests helped me come up with the right typings for the path module. Without the test and based just on the code, the argument `disableSpaceAfterFlags` would have been required, but in reality it's optional.

Or do you mean to only exclude "legacy" tests in the `test` folder?

Reply:
Ok then, keep it